src/hp/magicsock/endpoint.rs

```diff
@@ -560,15 +559,15 @@ impl Endpoint {
                     let ep = sp.to;
                     let region_id = usize::from(ep.port());
                     // FIXME: this creates a deadlock as it needs to interact with the run loop in the conn::Actor
-                    let region_code = self.get_derp_region(region_id).await.map(|r| r.region_code);
+                    // let region_code = self.get_derp_region(region_id).await.map(|r| r.region_code);
```

Comment:
Why is this commented out? We don't want to deal with regions yet?

Reply:
because of the `FIXME` above, I haven't figured out a clean way yet, and this is only for CLI pings, which we currently don't thread through yet.